packages/cashier/src/Components/Receipt/payment-agent-receipt.jsx

```diff
@@ -37,26 +36,33 @@ class PaymentAgentReceipt extends React.Component {
                         </span>
                     </div>
                     <div className='cashier__transferred-details-wrapper'>
-                        <Icon icon={`IcCurrency-${currency_lowercase}`} />
                         <span className='cashier__transferred-details'>
-                            <span className='cashier__text--bold'>{getCurrencyDisplayCode(currency)}</span>
-                            &nbsp;(
-                            {loginid})
+                            <span className='cashier__text--bold cashier__text--end'>
+                                {getCurrencyDisplayCode(currency)}
+                            </span>
+                            <span className='cashier__text--sub'>{loginid}</span>
                         </span>
+                        <Icon size={32} icon={`IcCurrency-${currency_lowercase}`} />
                         <Icon className='cashier__transferred-icon' icon='IcArrowLeftBold' />
-                        <Icon icon='IcPA' />
+                        <Icon size={32} icon='IcPA' />
                         <span className='cashier__transferred-details'>
                             {receipt.payment_agent_name && (
-                                <span className='cashier__text--bold'>{receipt.payment_agent_name}&nbsp;</span>
+                                <span className='cashier__text--bold'>{receipt.payment_agent_name}</span>
                             )}
-                            ({receipt.payment_agent_id})
+                            <span className='cashier__text--sub'>{receipt.payment_agent_id}</span>
                         </span>
                     </div>
                 </div>
-                <h2 className='cashier__header'>
+                <h2 className='cashier__header cashier__header-large'>
```

Comment:
you can use text components here or remove cashier__header class because both of this class only change a font size
```suggestion
                <h2 className='cashier__header-large'>
```
```suggestion
                <Text as="h1">

Reply:
ditto